src/metax_api/tests/api/rest/base/views/files/read.py

```diff
@@ -44,6 +44,18 @@ def test_read_file_list_filter_by_project(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data['count'], file_count)
 
+    def test_read_file_list_filter_by_project_and_path(self):
+        proj = File.objects.get(pk=1).project_identifier
+        path = "/project_x_FROZEN/Experiment_X/Phase_1/2017/01"
+        file_count = File.objects.filter(file_path__contains=path).count()
```

Comment:
Works by god's grace right now. To be sure, could use project filter here additionally, since the below query also actually needs the project parameter.